### PR TITLE
terraform-docs 0.4.5

### PR DESCRIPTION
**Pull request by metmajer**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/terraform-docs.rb [hunk 1]
```diff
@@ -1,82 +1,24 @@
-require "language/go"
-
 class TerraformDocs < Formula
   desc "Tool to generate documentation from Terraform modules"
   homepage "https://github.com/segmentio/terraform-docs"
-  url "https://github.com/segmentio/terraform-docs/archive/v0.3.0.tar.gz"
-  sha256 "0cfac8ed50a6ba458ec5177e493fd8adc05395f3d9ba79504dc33ce6e5733fcd"
-  head "https://github.com/segmentio/terraform-docs.git"
-
-  bottle do
```
Comment (commitay) [inline Formula/terraform-docs.rb]: The bottle block should not be edited manually, please restore it.

File: Formula/terraform-docs.rb [hunk 2]
```diff
@@ -1,82 +1,24 @@
...
-      }
-      }
-
-      // The VPC ID.
-      output "vpc_id" {
-      value = "vpc-5c1f55fd"
-      }
-    EOS
-    system "#{bin}/terraform-docs", "json", testpath
+    system "#{bin}/terraform-docs", "--version"
```
Comment (commitay) [inline Formula/terraform-docs.rb]: Why is the test changing?

File: Formula/terraform-docs.rb (hunk 1 above)
Comment (metmajer) [reply Formula/terraform-docs.rb]: Thanks for the feedback. After going through the documentation, I had assumed that your Jenkins pipeline would automatically insert this block. The block has been restored with https://github.com/Homebrew/homebrew-core/pull/32799/commits/06479f68003572d26b6327aceb78dcd048972209.

File: Formula/terraform-docs.rb (hunk 2 above)
Comment (metmajer) [reply Formula/terraform-docs.rb]: Restored with https://github.com/Homebrew/homebrew-core/pull/32799/commits/ba07fb0f2646aa030a22eacad9259f92adaebe5e!

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@metmajer somehow I can't merge this without conflicts. Can you squash all your commits into a single one, with message `terraform-docs 0.4.5`?

**Comment by fxcoudert** [comment]
@metmajer yes the bottles are taken care of by the CI and maintainers when we merge. Thanks for your contribution to Homebrew!

**Pull request closed without merging**